package.json

```diff
@@ -287,11 +287,23 @@
             "default": true,
             "markdownDescription": "Use TLS over GRPC server interface (strongly recommended - only disable if you know what you are doing!!!)"
           },
-          "runme.terminal.enabled": {
+          "runme.terminal.backgroundTask": {
+            "type": "boolean",
+            "scope": "machine",
+            "default": true,
+            "markdownDescription": "Enable Runme integrated terminal in Background tasks (Experimental)"
```

Comment:
let's replace all mention of "integrated" with "notebook". VS Code talks about it's native terminal as "integrated terminal". Just so we can avoid confusion.

Reply:
Lowercase `experimental` just to be consistent with others

Reply:
Fixed